From 4cabeca505ab293ff53b35e5591abd5dfb1a114a Mon Sep 17 00:00:00 2001
From: Anand N Sunkad <asunka@codeaurora.org>
Date: Wed, 24 Jun 2015 20:50:17 +0530
Subject: [PATCH 07/11] wcnss_service: Migrate from QMI to QCCI

Use new QCCI APIs to communicate with the device
management service for getting the wlan mac address.

This is need because the older QMI APIs are getting
deprecated.

CRs-Fixed: 871051
Change-Id: Ia5c37ba8e86261c5e912c83f9988124b4355c5ec
---
 wcnss-service/Android.mk | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/wcnss-service/Android.mk b/wcnss-service/Android.mk
index 4af9ab2..e2f2d11 100644
--- a/wcnss-service/Android.mk
+++ b/wcnss-service/Android.mk
@@ -41,14 +41,13 @@ ifeq ($(strip $(TARGET_USES_QCOM_WCNSS_QMI)),true)
 ifneq ($(QCPATH),)
 include $(CLEAR_VARS)
 
-LOCAL_C_INCLUDES += $(TARGET_OUT_HEADERS)/qmi/inc
+LOCAL_C_INCLUDES += $(TARGET_OUT_HEADERS)/qmi-framework/inc
 LOCAL_C_INCLUDES += $(TARGET_OUT_HEADERS)/qmi/services
 LOCAL_C_INCLUDES += $(TARGET_OUT_HEADERS)/qmi/platform
-LOCAL_C_INCLUDES += $(TARGET_OUT_HEADERS)/qmi/src
-LOCAL_C_INCLUDES += $(TARGET_OUT_HEADERS)/qmi/core/lib/inc
+LOCAL_C_INCLUDES += $(TARGET_OUT_HEADERS)/qmi/inc
 
 LOCAL_SHARED_LIBRARIES := libc libcutils libutils liblog
-LOCAL_SHARED_LIBRARIES += libqmiservices libqmi libqcci_legacy libqmi_client_qmux
+LOCAL_SHARED_LIBRARIES += libqmiservices libqmi_cci libidl
 LOCAL_C_INCLUDES += $(TARGET_OUT_HEADERS)/libmdmdetect/inc
 LOCAL_SHARED_LIBRARIES += libmdmdetect
 
-- 
2.10.0

